rust/actyx/swarm/cli/Cargo.toml

```diff
@@ -8,23 +8,23 @@ authors = ["Actyx AG"]
 acto = "0.2.9"
 actyx_sdk = { path = "../../../sdk" }
 anyhow = "1.0.52"
-api = { path = "../../api" }
 async-process = "1.3.0"
-ax-futures-util = { path = "../../ax-futures-util/" }
+ax = { path = "../../ax" }
 cbor-data = "0.8.7"
 chrono = "0.4.19"
 crossbeam = "0.8.1"
-crypto = { path = "../../crypto" }
 futures = { version = "0.3.19", package = "futures" }
 hex = "0.4.3"
-ipfs-embed = { version = "0.26.0", default-features = false, features = ["tokio"] }
+ipfs-embed = { version = "0.26.0", default-features = false, features = [
```

Comment:
The `ax` version is 0.26.1, should we match them?

Reply:
I guess so

Reply:
@jmg-duarte

Reply:
Cargo does in fact use `0.26.1`, I'm not a fan of the behavior but it works indeed. I'll match them regardless